.ci/pipeline_definitions

```diff
@@ -36,9 +36,9 @@ autoscaler:
             dockerfile: './cluster-autoscaler/Dockerfile'
     steps:
       test:
-        image: 'golang:1.20.5'
+        image: 'golang:1.21.6'
```

Comment:
Can you change it to `1.22.2` instead of `1.21.6`?

Reply:
the go version in CA 1.29 is still 1.21, so not sure if this will work out.

Reply:
done